src/reducers/boxes.js

```diff
@@ -53,6 +53,8 @@ const boxesReducer = (prevState = initialState, action) => {
       return prevState.clear().merge(action.newLayout);
     case SAVE:
       return prevState;
+    case CLEAR_ALL:
```

Comment:
have you tested this? might need to be return prevState.clear().mergeWith(initialState)